src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetPackageSearchService.cs

```diff
@@ -493,17 +492,28 @@ private async ValueTask<IInstalledAndTransitivePackages> GetInstalledAndTransiti
             {
                 if (isSolution)
                 {
+                    // We only need the installed packages? What about recommender
+                    IReadOnlyCollection<IPackageReferenceContextInfo> installedSolutionTabPackages = await GetAllInstalledPackagesAsync(projectContextInfos, cancellationToken);
+                    PackageCollection installedPackageCollection = PackageCollection.FromPackageReferences(installedSolutionTabPackages);
                     packageFeeds.mainFeed = new InstalledPackageFeed(installedPackageCollection, metadataProvider);
                 }
                 else // is Project
                 {
                     CounterfactualLoggers.PMUITransitiveDependencies.EmitIfNeeded();
                     if (await ExperimentUtility.IsTransitiveOriginExpEnabled.GetValueAsync(cancellationToken))
                     {
+                        // We need installed and Transitive Packages
+                        IInstalledAndTransitivePackages installedTabWithTransitiveDepsPackages = await GetInstalledAndTransitivePackagesAsync(projectContextInfos, useTransitiveOrigins: true, cancellationToken);
```

Comment:
I think we can save some characters by removing the "Deps" part of the variable name, which I believe is short for Dependencies. I don't think it adds any clarity to the variable name.

```suggestion
                        IInstalledAndTransitivePackages installedTabWithTransitivePackages = await GetInstalledAndTransitivePackagesAsync(projectContextInfos, useTransitiveOrigins: true, cancellationToken);
```

Reply:
Renamed to `installedTabWithTransitiveOrigins`